src/components/AdminPanel/MedCentersForm/FormValidationConst.ts

```diff
@@ -0,0 +1,13 @@
+export const Required = {
+  required: {
+    value: true,
+    message: 'Required to fill.',
+  },
+};
```

Comment:
don't you already have such configuration for other forms?

Reply:
I use one for all forms